contracts/lifebankcoin/lifebankcoin.abi

```diff
@@ -126,27 +158,32 @@
         {
             "name": "clear",
             "type": "clear",
-            "ricardian_contract": "---\nspec-version: 0.1.0\ntitle: Clear table\nsummary: The intent of the `{{ clear }}` action is to delete all table records and is used for development purposes on test-nets and must not be published to mainnet.\nicon: https://lifebank.io/icons/admin.png"
+            "ricardian_contract": ""
```

Comment:
@leisterfrancisco  seems you are deleting all the ricardian clauses , please recompile contracts with clauses